core/optaplanner-core-impl/src/main/java/org/optaplanner/core/api/domain/variable/AbstractVariableListener.java

```diff
@@ -23,17 +23,6 @@
  */
 public interface AbstractVariableListener<Solution_, Entity_> extends Closeable {
 
-    /**
-     * When set to {@code true}, this has a slight performance loss in Planner.
-     * When set to {@code false}, it's often easier to make the listener implementation correct and fast.
-     *
-     * @return true to guarantee that each of the before/after methods will only be called once per entity instance
-     *         per operation type (add, change or remove).
-     */
-    default boolean requiresUniqueEntityEvents() {
-        return false;
-    }
-
```

Comment:
I presume this is not a real BC change because the only interface they could implement is VariableListener